configuration/helm-charts/ksqldb.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Integrating ksqlDB with Kafbat UI
```

Comment:
I believe this can be moved out of helm-charts folder as this brings the overall knowledge about setting up ksql

Reply:
Moved under the more relevant directory `configuration` ([5e1ea8b](https://github.com/kafbat/ui-docs/pull/26/commits/5e1ea8bb846029d76de44a350a2c50fff6b1a32d))